regimes/gr/gr.go

```diff
@@ -0,0 +1,83 @@
+// Package gr provides the tax region definition for Greece.
+package gr
+
+import (
+	"github.com/invopop/gobl/bill"
+	"github.com/invopop/gobl/cbc"
+	"github.com/invopop/gobl/currency"
+	"github.com/invopop/gobl/i18n"
+	"github.com/invopop/gobl/l10n"
+	"github.com/invopop/gobl/regimes/common"
+	"github.com/invopop/gobl/tax"
+)
+
+func init() {
+	tax.RegisterRegime(New())
+}
+
+// Custom keys used typically in meta or codes information.
+const (
+	KeyIAPRVATCategory cbc.Key = "iapr-vat-category"
+)
```

Comment:
Comments below, but I think this one should be removed and we can use the extension key.

Reply:
Addressed ins [512ab72](https://github.com/invopop/gobl/pull/307/commits/512ab72a85133320e6986ac3f1e59dc2fab0285c)